tests/base.py

```diff
@@ -42,17 +42,19 @@ class TestSquareBase(ABC, TestCase):
         STATIC_START_DATE = "2020-07-13T00:00:00Z"
         START_DATE = ""
         PRODUCTION_ONLY_STREAMS = {'bank_accounts', 'payouts'}
+        sandbox_test_name = "tap_tester_square_sandbox_tests"
+        prod_test_name = "tap_tester_square_prod_tests"
```

Comment:
Since these value are constants, those shold be capatalised.

```suggestion
        TEST_NAME-SANDBOX = "tap_tester_square_sandbox_tests"
        TEST_NAME_PROD = "tap_tester_square_prod_tests"
```